src/parser/parsers.h

```diff
@@ -1373,11 +1375,7 @@ template<typename Ctx> Result<> makeArrayNewData(Ctx& ctx, Index pos) {
 }
 
 template<typename Ctx> Result<> makeArrayNewElem(Ctx& ctx, Index pos) {
-  auto type = typeidx(ctx);
-  CHECK_ERR(type);
-  auto data = dataidx(ctx);
-  CHECK_ERR(data);
-  return ctx.makeArrayNewElem(pos, *type, *data);
+  return ctx.in.err("unimplemented instruction");
```

Comment:
Why is this now unimplemented?

Reply:
I noticed that it incorrectly used `dataidx` above, when it should have used `elemidx`, which doesn't exist yet. I removed the incorrect implementation with the standard "unimplemented" error.